data/nation/donetsk.ruleset

```diff
@@ -0,0 +1,111 @@
+[nation_donetsk]
+
+name=_("Donetsk")
+plural=_("?plural:Donchyani")
+groups="Modern", "European"
+legend=_("The Donetsk People's Republic is an unrecognised republic of\
+ Russia in the occupied parts of eastern Ukraine's Donetsk Oblast, with\
+ its capital in Donetsk. The DPR was created by Russian-backed\
+ aramilitaries in 2014, and it initially operated as a breakaway state\
+ Until it was annexed by Russia in 2022. Following Ukraine's Revolution\
+ of Dignity in 2014, pro-Russian, counter-revolutionary unrest erupted\
+ in the eastern part of the country. Russia annexed Crimea from Ukraine,\
+ while the armed separatists seized government buildings and proclaimed\
+ the Donetsk People's Republic and Luhansk People's Republic as independent\
+ states. This sparked the war in Donbas, part of the wider Russo-Ukrainian\
+ War. The DPR and LPR are sometimes described as puppet states of Russia\
+ during this conflict. They received no international recognition from\
+ United Nations member states before 2022. On 21 February 2022, Russia\
+ recognised the DPR and LPR as sovereign states. Three days later, Russia\
+ launched a full-scale invasion of Ukraine, partially under the pretext of\
+ protecting the republics. Russian forces captured more of Donetsk Oblast,\
+ which became part of the DPR. In September 2022, Russia proclaimed the\
+ annexation of the DPR and other occupied territories, following\
+ illegitimate referendums. The United Nations General Assembly passed a\
+ resolution calling on countries not to recognise what it called the\
+ "attempted illegal annexation" and demanded that Russia "immediately,\
+ completely and unconditionally withdraw.")
```

Comment:
I think this description is too closely related to current events. Since it will likely not be revised in the next decade, try to write something more generic.